### PR TITLE
bpo-31801:  Enum:  add _ignore_ as class option

### PR DESCRIPTION
**Pull request by ethanfurman**
_ignore_ is a list, or white-space seperated str, of names that will not
be candidates for members; these names, and _ignore_ itself, are removed
from the final class.

https://bugs.python.org/issue31801

### CODE REVIEW

**Comment by csabella** [comment]
Should this be added to the documentation?

**Comment by ethanfurman** [comment]
On 01/18/2018 05:05 PM, Cheryl Sabella wrote:
> Should this be added to the documentation?
Yes it should.  Thanks for the reminder.

File: Doc/library/enum.rst [hunk 1]
```diff
@@ -994,6 +1014,9 @@ Supported ``_sunder_`` names
 
 - ``_missing_`` -- a lookup function used when a value is not found; may be
   overridden
+- ``_ignore_`` -- a list of names, either as a :func:`list` or a :func:`str`,
```
Comment (csabella) [inline Doc/library/enum.rst:1017]: Should it be added to the `version added` section just below this?

Comment (ethanfurman) [reply Doc/library/enum.rst:1017]: Done.

**Pull request merged**